stable/kommander-karma/templates/federated-addon.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: types.kubefed.io/v1beta1
```

Comment:
yes, let's create a flag like https://github.com/mesosphere/charts/blob/dev/stable/kommander/values.yaml#L52 and wrap the federated resources under their respective flags as done in kommander.